tools/tools.go

```diff
@@ -0,0 +1,8 @@
+//go:build tools
+
+package tools
+
+import (
+	_ "github.com/psampaz/go-mod-outdated"
```

Comment:
what is go mod outdated?

Reply:
Shows you out dated modules in your go.mod file.  Pretty handy if you like to keep that sort of thing up to date.  Pretty annoying if you don't.  In this case, its only used as part of the tools module.